js/ndax.js

```diff
@@ -14,34 +14,58 @@ module.exports = class ndax extends Exchange {
         return this.deepExtend (super.describe (), {
             'id': 'ndax',
             'name': 'NDAX',
-            'countries': [ 'US' ], // United States
+            'countries': [ 'CA' ], // Canada
             'rateLimit': 1000,
             'pro': true,
             'has': {
-                'withdraw': true,
+                'spot': true,
+                'margin': false,
+                'swap': false,
+                'future': false,
+                'option': false,
                 'cancelAllOrders': true,
                 'cancelOrder': true,
                 'createDepositAddress': true,
                 'createOrder': true,
                 'editOrder': true,
                 'fetchAccounts': true,
                 'fetchBalance': true,
+                'fetchBorrowRate': false,
+                'fetchBorrowRateHistory': false,
+                'fetchBorrowRates': false,
                 'fetchCurrencies': true,
                 'fetchDepositAddress': true,
                 'fetchDeposits': true,
+                'fetchFundingHistory': false,
+                'fetchFundingRate': false,
+                'fetchFundingRateHistory': false,
+                'fetchFundingRates': false,
+                'fetchIndexOHLCV': false,
+                'fetchIsolatedPositions': false,
                 'fetchLedger': true,
+                'fetchLeverage': false,
                 'fetchMarkets': true,
+                'fetchMarkOHLCV': false,
                 'fetchMyTrades': true,
                 'fetchOHLCV': true,
                 'fetchOpenOrders': true,
                 'fetchOrder': true,
                 'fetchOrderBook': true,
                 'fetchOrders': true,
                 'fetchOrderTrades': true,
+                'fetchPositions': false,
+                'fetchPositionsRisk': false,
+                'fetchPremiumIndexOHLCV': false,
                 'fetchTicker': true,
                 'fetchTrades': true,
                 'fetchWithdrawals': true,
+                'loadLeverageBrackets': false,
+                'reduceMargin': false,
+                'setLeverage': false,
+                'setPositionMode': false,
                 'signIn': true,
+                'transferOut': false,
```

Comment:
Why would we include `transferOut` and not `transferIn` nor `transfer` ?

Reply:
We would include `transfer`, I just know that `transferOut` would be false because it's a 1 to 1 relationship type of method where there can only be 1 accunt to transfer to, so it's reserved for classes like `binanceusdm` or `kucionfutures` where transfering out means going from futures to main, or futures to spot. I haven't looked into ndax sub accounts in depth, but I know that since we're working with the main account, that we are not in this type of situation

Reply:
I don't think that `transferOut` is a part of the unified API though. We will eventually run into problems with it due to conflicts with the type of the destination "funding" account between different exchanges.

Reply:
Ok, I opened a PR to take it out of has https://github.com/ccxt/ccxt/pull/11337